feat: Adding `cosmovisor version` command (#10103)
cosmos/cosmos-sdk: cosmovisor/CHANGELOG.md

+ [\#8590](https://github.com/cosmos/cosmos-sdk/pull/8590) File watcher for cosmovisor. Instead of parsing logs from stdin and stderr, we watch the `<DAEMON_HOME>/data/upgrade-info.json` file updates using polling mechanism.
+ [\#10128](https://github.com/cosmos/cosmos-sdk/pull/10128) Change default value of `DAEMON_RESTART_AFTER_UPGRADE` to `true`.

> + [\#9999](https://github.com/cosmos/cosmos-sdk/issues/9999) Added `version` command which returns the version of the cosmovisor along with the version of the application.

Reviewer: ```suggestion
+ [\#9999](https://github.com/cosmos/cosmos-sdk/issues/9999) Added `version` command that returns the cosmovisor version and the application version.
```
